migrations/0012_doctorateadmission_fk_thesis_institute.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.2.24 on 2021-12-14 15:22
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('admission', '0011_auto_20211216_1514'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='doctorateadmission',
+            name='thesis_institute',
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name='+',
+                to='base.EntityVersion',
+                verbose_name='Thesis institute',
+            ),
+        ),
+        migrations.AddField(
+            model_name='doctorateadmission',
+            name='other_thesis_location',
+            field=models.CharField(
+                blank=True,
+                default='',
+                max_length=255,
+                verbose_name='Other thesis location'
+            ),
+        ),
+    ]
```

Comment:
Tes tests unitaires plantent suite à cette migration (il ne peut pas convertir un CharField en FK IntegerField), je te propose de faire les opérations suivante "à la mano" :

1. tu renommes (`migrations.RenameField`) le champ `thesis_institute` en `other_thesis_location`
2. tu crée le nouveau champ `thesis_institute` (et donc les valeurs pour les lignes existantes seront nulles) 

```suggestion
    operations = [
        migrations.RenameField(
            model_name='doctorateadmission',
            old_name='thesis_institute',
            new_name='other_thesis_location',
        ),
        migrations.AddField(
            model_name='doctorateadmission',
            name='thesis_institute',
            field=models.ForeignKey(
                blank=True,
                null=True,
                on_delete=django.db.models.deletion.SET_NULL,
                related_name='+',
                to='base.EntityVersion',
                verbose_name='Thesis institute',
            ),
        ),
    ]
```

Il se peut que ça ne fonctionne pas car il execute tout ça dans une seule transaction, si c'est le cas il faut splitter les opérations en deux migrations.